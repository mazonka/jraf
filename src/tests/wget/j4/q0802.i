command=jw md 0 /.jraf.sys/users : login admin@server.com * : login admim@server.com * : put f82d3bb17612458c /z 0 1 Kg== : read 5aeb1f6c5cd9c44a /z