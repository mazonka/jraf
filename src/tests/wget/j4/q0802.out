OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : OK http://127.0.0.1:16000/jraf? : OK 1 : OK 1 1 Kg==
==
